drivers/mma7660/include/mma7660_params.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2016 University of California, Berkeley
+ * Copyright (C) 2016 Michael Andersen <m.andersen@cs.berkeley.edu>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     drivers_mma7660
+ * @{
+ *
+ * @file
+ * @brief       Default configuration for MMA7660 accelerometer.
+ *
+ * @author      Michael Andersen <m.andersen@cs.berkeley.edu>
+ *
+ */
+
+#ifndef MMA7660_PARAMS_H
+#define MMA7660_PARAMS_H
+
+#include "board.h"
+#include "mma7660.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Set default configuration parameters for MMA7660 devices
```

Comment:
Wait, does that mean that the [HDC1000](https://github.com/RIOT-OS/RIOT/blob/master/drivers/hdc1000/include/hdc1000_params.h#L30-L47), [LIS3DH](https://github.com/RIOT-OS/RIOT/blob/master/drivers/lis3dh/include/lis3dh_params.h#L30-L59), [mma8x6x](https://github.com/RIOT-OS/RIOT/blob/master/drivers/mma8x5x/include/mma8x5x_params.h#L30-L61) etc etc are all wrong?